Cargo.toml

```
@@ -1,6 +1,16 @@
 [workspace]
 resolver = "2"
-members = ["diplomat-gen", "bakeddata", "provider", "temporal_capi", "depcheck", "zoneinfo", "zoneinfo-test-gen"]
+members = [
+    "provider",
+    "temporal_capi",
+    "zoneinfo",
+
+    # Tools
+    "tools/bakeddata",
+    "tools/depcheck",
+    "tools/diplomat-gen",
+    "tools/zoneinfo-test-gen",
```

Maybe just use an asterisk specifier?

```suggestion
    "tools/*",
```